src/proxy-ui-api/frontend/src/router.ts

```diff
@@ -31,10 +31,12 @@ import CertificateDetails from '@/views/CertificateDetails/CertificateDetails.vu
 import Service from '@/views/Service/Service.vue';
 import GenerateCertificateSignRequest from '@/views/GenerateCertificateSignRequest/GenerateCertificateSignRequest.vue';
 import store from '@/store';
-import {Permissions, RouteName} from '@/global';
+import { Permissions, RouteName } from '@/global';
 import ServiceEndpoints from '@/views/Service/Endpoints/Endpoints.vue';
 import ServiceParameters from '@/views/Service/Parameters/ServiceParameters.vue';
+import InternalCertificateDetails from '@/views/InternalCertificateDetails/InternalCertificateDetails.vue';
 
+type Next = Parameters<NavigationGuard>[2];
```

Comment:
This could be commented a bit. I guess this is just type declaration for routers next object, but could use a comment.

Reply:
Yes, this looks quite cryptic. Added a comment.